clients/python/tests/conftest.py

```diff
@@ -47,47 +61,57 @@ def uc_server():
             pass
 
 
-@pytest.fixture(scope="session")
-def api_client():
-    config = unitycatalog.Configuration(
-        host = "http://localhost:8081/api/2.1/unity-catalog"
-    )
+@pytest_asyncio.fixture()
```

Comment:
Note on the session scoping: pytest-asyncio has a restriction in its implementation for 0.22.0 and above where session scoped fixtures are not supported due to a decision to not support handling closures in futures. 
The recommendation is to move to function-based fixtures, which I've done throughout the suites (I validated that session based fixtures, even with setting configurations in pytest.ini, do not function correctly with the pytest execution environment).